pkg/rehearse/jobs_test.go

```diff
@@ -640,7 +598,7 @@ func TestWaitForJobs(t *testing.T) {
 				return true, w, nil
 			})
 
-			executor := NewExecutor(nil, 0, "", &pjapi.Refs{}, true, loggers, cs.ProwV1().ProwJobs("test"))
+			executor := NewExecutor(nil, 0, "", &pjapi.Refs{}, false, loggers, cs.ProwV1().ProwJobs("test"))
```

Comment:
What's the meaning of this?

Reply:
Oops, I didn't again :). I include these changes in #87 . I will remove them from this PR.